Demo/Demo/FeatureViewControllers/NativeCheckoutDemoViewController.swift

```diff
@@ -40,8 +60,16 @@ class NativeCheckoutDemoViewController: FeatureBaseViewController {
     @objc func didTapNativeCheckoutButton() {
         Task {
             nativeCheckoutButton.startAnimating()
-            try await baseViewModel.checkoutWithNativeClient()
+            viewModel.checkoutWithOrderID()
             nativeCheckoutButton.stopAnimating()
         }
     }
+
+    @objc func didTapNativeCheckoutWithOrderButton() {
```

Comment:
This method does the same as the one above. Can we delete it? Also, we don't need two PayPal Native buttons anymore right?

Reply:
yeah you are right. This viewcontroller doesnt work any way, because i focused on making it in Swift UI. Once all of this is merged, the idea is to delete all the view controllers